src/main/java/org/fogbowcloud/saps/engine/core/dispatcher/SubmissionDispatcher.java

```diff
@@ -238,13 +241,18 @@ else if (rowAux.length() == 2)
 	 * @param inputdownloadingPhaseTag inputdownloading phase tag
 	 * @param preprocessingPhaseTag    preprocessing phase tag
 	 * @param processingPhaseTag       processing phase tag
+	 * @param digestProcessing
+	 * @param processingPhaseTag2
```

Comment:
what is a `processingPhaseTag2`. In fact, we need to add *good* docs related to this tag and digest stuff. I mean, with some examples. This is non-trivial stuff (so, need proper doc).

As I commented before, the code you added looks good to me but we need to refactor legacy code (you can make it in another PR):
1. remove silly documentation (e.g. "This function adds new user." in a method called `addNewUserInCatalog` does not add much, so remove the setence)
2. remove dead code
3. overall, we have important code in classes name *Util (e.g. CatalogUtils) and ancillary code in the main classes (e.g. formatPathRow)

Reply:
I created an other issue (#74) reporting on.